lib/wise_homex/models/ready_meter.ex

```diff
@@ -0,0 +1,19 @@
+defmodule WiseHomex.ReadyMeter do
+  @moduledoc false
+
+  use WiseHomex.BaseModel
+
+  embedded_schema do
+    belongs_to :ready_installation, WiseHomex.ReadyInstallation
+    belongs_to :device, WiseHomex.Device
+
+    field :external_id, :integer
+    field :consumptiontype, :string
```

Comment:
Shouldn't this be `consumption_type` for consistency in the API? Or is it already merged? :thinking: 
(The same for `serialnumber` and `metertype` below....)

Reply:
Or is it Kamstrup-specific field names?

Reply:
I've taken the attributes directly from Kamstrup.

Reply:
@skovmand Really good feedback. I've changed it both here and on server side.